lib/src/client.dart

```diff
@@ -101,6 +115,7 @@ class ClientImpl implements Client {
   ClientImpl(this._url, this._config, this._transportBuilder) {
     _token = _config.token;
     _data = _config.data;
+    _headers = _config.headers;
   }
```

Comment:
```dart
ClientImpl(this._url, this._config, this._transportBuilder)
  : _token = _config.token,
    _data = _config.data,
    _headers = Map<String, Object?>.of(_config.headers);
 ```
 
 You can set it before constructor's body.
 And optionally make a copy of hash map.